demos/demos_databases_apis/azure_data_explorer/graphistry_ADX_kusto_demo.ipynb

```diff
@@ -0,0 +1,469 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c80d3a57-7c78-4bc3-bb4d-6b8cbd71f9fc",
+   "metadata": {},
+   "source": [
+    "# Persistent Graph Semantics in Azure Data Explorer & Graphistry Integration\n",
```

Comment:
Maybe: `"...  Demo: Redteam Logins"` ?

Reply:
oof feels like a mouthful

```Tutorial: Manage graphs with Kusto Persistent Graph & Graphistry``` ? ADE feels like a detail here. Mentioning in regular areas of ipynb, including top, will still hit SEO.